TestKeyspaceGroupManagerTestSuite/TestLoadKeyspaceGroupsAssignment timeout

## Flaky Test

### Which jobs are failing
```
2023-04-22T01:58:49.9607745Z --- FAIL: TestKeyspaceGroupManagerTestSuite (60.54s)
2023-04-22T01:58:49.9608260Z     --- FAIL: TestKeyspaceGroupManagerTestSuite/TestLoadKeyspaceGroupsAssignment (34.78s)
2023-04-22T01:58:49.9608563Z         keyspace_group_manager_test.go:593:
2023-04-22T01:58:49.9609184Z                    Error Trace:    /home/runner/work/pd/pd/pkg/tso/keyspace_group_manager_test.go:593
2023-04-22T01:58:49.9610407Z                                                            /home/runner/work/pd/pd/pkg/tso/keyspace_group_manager_test.go:135
2023-04-22T01:58:49.9610724Z                    Error:          Received unexpected error:
2023-04-22T01:58:49.9611552Z                                    [PD:tso:ErrLoadKeyspaceGroupsTerminated]load keyspace groups terminated
2023-04-22T01:58:49.9612241Z                    Test:           TestKeyspaceGroupManagerTestSuite/TestLoadKeyspaceGroupsAssignment
2023-04-22T01:58:49.9612325Z FAIL
```

### CI link
https://github.com/tikv/pd/actions/runs/4770285461/jobs/8481414330?pr=6358

### Reason for failure (if possible)
This test tries to initialize and load 4906 keyspace group with etcd from etcd and sometimes CI job takes more 30 seconds and cause tso.KeyspaceGroupManager to timeout.

### Anything else

--- a/pkg/tso/keyspace_group_manager_test.go
+++ b/pkg/tso/keyspace_group_manager_test.go
@@ -125,7 +125,7 @@ func (suite *keyspaceGroupManagerTestSuite) TestNewKeyspaceGroupManager() {
 // TestLoadKeyspaceGroupsAssignment tests the loading of the keyspace group assignment.
 func (suite *keyspaceGroupManagerTestSuite) TestLoadKeyspaceGroupsAssignment() {
 	re := suite.Require()
-	maxCountInUse := int(mcsutils.MaxKeyspaceGroupCountInUse)
+	maxCountInUse := 512
 	// Test loading of empty keyspace group assignment.
 	suite.runTestLoadKeyspaceGroupsAssignment(re, 0, 0, 100)
 	// Test loading of single keyspace group assignment.

